package-testing/android-relay/app/src/main/java/cloud/eppo/android/sdkrelay/EppoSdkRelay.kt

```diff
@@ -0,0 +1,27 @@
+package cloud.eppo.android.sdkrelay
+
+import android.app.Application
+import android.util.Log
+import cloud.eppo.android.EppoClient
+
+class EppoSdkRelay : Application() {
+
+  val API_KEY: String = BuildConfig.API_KEY // Set in root-level local.properties
+
+  override fun onCreate() {
+    super.onCreate()
+    EppoClient.Builder(API_KEY, this)
+        .isGracefulMode(false)
+        .assignmentLogger { assignment ->
+          Log.d(
+              TAG,
+              "${assignment.experiment}-> subject: ${assignment.subject} assigned to ${assignment.experiment}")
+        }
+        .buildAndInitAsync()
+        .thenAccept { Log.d(TAG, "Eppo SDK initialized") }
+  }
+
+  companion object {
```

Comment:
![image](https://github.com/user-attachments/assets/b996db40-8cc6-49d8-8da3-8d347f03eef3)

Reply:
lol. it's hard not to have fun when using kotlin